helm/g2p-sandbox/values.yaml

```diff
@@ -208,7 +208,7 @@ ph-ee-engine:
     hostname: "channel.sandbox.mifos.io"
     stub_hostname: "channel-gsma.sandbox.mifos.io"
     SPRING_PROFILES_ACTIVE: "bb,tenants"
-    DFSPIDS: "rhino,gorilla"
+    DFSPIDS: "rhino,gorilla,lion"
```

Comment:
check whether this can be added from global params

Reply:
This is not being picked up global params